Core/Announcements.lua

```diff
@@ -18,27 +18,37 @@ function Announcements:AnnounceAttemptForItem(item)
 		local displayedText
 		local attempts = item.attempts or 1
 		local total = item.attempts or 1
+		local chance
+                if item.method == CONSTANTS.DETECTION_METHODS.COLLECTION then
+                        chance = (item.attempts or 0) / (item.chance or 100)
+                        if chance < 0 then chance = 0 end
+                        if chance > 1 then chance = 1 end
+                        chance = chance * 100
+                else
+                        local dropChance = Rarity.Statistics.GetRealDropPercentage(item)
+                        chance = 100 * (1 - math.pow(1 - dropChance, attempts))
+		end
```

Comment:
This seems like something we could easily test, but to do that it should likely be moved to a separate function?

I'll take care of the tests if you could extract it; the function can just live in the `Announcements` module for now :)

Reply:
I guess we can always do that later, if needed.